DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<p>Maybe this cell should be split into 4, one after printing the query, another one for running the job, another one for getting the table, and another one for deleting the variables.</p><p> </p><p>And after splitting the cell, split the previous cell with the explanation and put each corresponding part of the text before the new cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done.</p>